code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -24,6 +24,11 @@
 		else if (light_amount < SHADOW_SPECIES_LIGHT_THRESHOLD) //heal in the dark
 			H.heal_overall_damage(1,1)
 
+/datum/species/shadow/check_roundstart_eligible()
+	if(SSevents.holidays && SSevents.holidays[HALLOWEEN])
+		return TRUE
+	else
+		..()

```

Comment:
`return ..()`

Reply:
actually, for all of these ifs, you don't even need an `else`, just go straight to return

Reply:
what's called a dangling else I believe

Reply:
`return (SSevents.holidays && SSevents.holidays[HALLOWEEN]) || ..()`

Reply:
imo an if is saner to read than doing this

Reply:
idk I think any boolean expression should be written as one